at-tanvo/Exercise3.rb

```diff
@@ -0,0 +1,42 @@
+class HtmlToText
+  def get_html(html)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/fbf4624d275e77d49e2a1ea2e742a839b9974143/at-tanvo/Exercise3.rb#L2"></span>Metrics/PerceivedComplexity: Perceived complexity for get_html is too high. [10/8]</td>
    </tr>
  </tbody>
</table>